cli/main.go

```diff
@@ -4,7 +4,6 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
-	"strings"
```

Comment:
I guess this is something `gofmt` did?

Reply:
`strings.Join` call was removed from the code ->

Reply:
Go build routine is not happy when module imported, but not used:
```
Step 7/32 : RUN go build -o ./bin/xk github.com/exekube/exekube/cli
 ---> Running in f6220d554507
# github.com/exekube/exekube/cli
src/github.com/exekube/exekube/cli/main.go:7:2: imported and not used: "strings"
ERROR: Service 'google' failed to build: The command '/bin/sh -c go build -o ./bin/xk github.com/exekube/exekube/cli' returned a non-zero code: 2
```